README.md

```diff
@@ -2,15 +2,18 @@
 
 [![Join the chat at https://gitter.im/deviantony/docker-elk](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/deviantony/docker-elk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Run the latest version of the ELK (Elasticsearch, Logstash, Kibana) stack with Docker and Docker-compose.
+Run the latest version of the ELK (Elasticsearch, Logstash, Kibana) stack with Docker and Docker Compose.
+Also includes Logspout which pipes Docker Log output into the ELK stack.
```

Comment:
I do not think that Logspout should be part of the 'core' elements of the stack, IMO it should be part of available extensions.

Should we introduce a new **Extensions** section in the README with the extra information about what brings the extension and how to start it within the stack?